packages/net-reactor-slayer.vm/net-reactor-slayer.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>net-reactor-slayer.vm</id>
+    <version>6.4.0.0</version>
```

Comment:
@emtuls, @mr-tz, the linter complains because the version does not follow [our documentation](https://github.com/mandiant/VM-Packages/wiki/Package-Structure#metapackages): 

> If the tool's version uses the 4th segment (e.g., version="[2.0.1.20210213]") update the metapackage's version to use the current date (YYYYMMDD) in the 4th segment (i.e., <version>2.0.1.20220224</version>)

The version should be `6.4.0.20230524`. I'll update the issue and retriger the bot